client/commonFramework/step-challenge.js

```diff
@@ -125,7 +125,7 @@ window.common = (function({ $, common = { init: [] }}) {
         // assume api returns string when fails
         return $el.parent()
           .find('.disabled')
-          .replaceWith('<p>' + data + '</p>');
+          .replaceWith('<p class="col-sm-4 col-xs-12">' + data + '</p>');
```

Comment:
@hangaebal Refer the comments left on the original thread for the expected fix, once you do, make sure you test your fix manually. By testing it.